packages/frontend/src/providers/ActiveJobProvider.tsx

```diff
@@ -45,16 +45,16 @@ export const ActiveJobProvider: FC<React.PropsWithChildren<{}>> = ({
     const { showToastSuccess, showToastError, showToastInfo } = useToaster();
 
     const toastJobStatus = useCallback(
-        (job: Job | undefined) => {
+        async (job: Job | undefined) => {
             if (!job || isJobsDrawerOpen) return;
 
             const toastTitle = jobStatusLabel(job?.jobStatus);
 
             switch (job.jobStatus) {
                 case 'DONE':
                     if (job.jobType === JobType.CREATE_PROJECT) {
-                        queryClient.invalidateQueries(['projects']);
-                        queryClient.invalidateQueries([
+                        await queryClient.invalidateQueries(['projects']);
+                        await queryClient.invalidateQueries([
```

Comment:
I feel like refetches can be ignored but not invalidations. what do you all think?

Reply:
Generally agreed, invalidations can have more important knock-on effects.